lib/agenda/schedule.js

```diff
@@ -3,32 +3,28 @@ const debug = require('debug')('agenda:schedule');
 
 /**
  * Schedule a job or jobs at a specific time
+ * @name Agenda#schedule
+ * @function
  * @param {String} when when the job gets run
- * @param {*} names array of job names to run
+ * @param {Array<String>} names array of job names to run
  * @param {Object} data data to send to job
- * @param {Function} cb called when schedule fails or passes
- * @returns {*} job or jobs created
+ * @returns {Promise<Job>} job or jobs created
```

Comment:
Would this not be a "promise with a Job" or "an array of promises each resolving a job"? I think that's why this was using a `*`.